tests/integration/nm/bond_test.py

```diff
@@ -76,10 +76,23 @@ def _bond_interface(name, options):
 
 
 @nmclient_context
-def _get_bond_current_state(name):
+def _get_bond_current_state(name, option=None):
     nmdev = nm.device.get_device_by_name(name)
     nm_bond_info = nm.bond.get_bond_info(nmdev) if nmdev else {}
-    return _convert_slaves_devices_to_iface_names(nm_bond_info)
+    if not nm_bond_info:
+        return {}
+    state = {
```

Comment:
nm_bond_info should include this state, why are you re-building it?

Reply:
We are using assert instead of `assert_state_match()`, we have to remove uninterested bond options.

I will amend the `_bond_interface` context to return a full state.

Reply:
I don't want to duplicate myself. The same thing has been done in integration test.
I will explain with comment why I rebuild the state.